packages/core/src/RenderingEngine/RenderingEngine.ts

```diff
@@ -626,7 +626,18 @@ class RenderingEngine implements IRenderingEngine {
       const prevCamera = vp.getCamera();
       const rotation = vp.getRotation();
       const { flipHorizontal } = prevCamera;
-      vp.resetCamera();
+      const resetPan = true;
+      const resetZoom = true;
+      const resetToCenter = true;
+      const resetRotation = false;
+      const supressEvents = true;
+      vp.resetCamera(
+        resetPan,
+        resetZoom,
+        resetToCenter,
+        resetRotation,
```

Comment:
i don't see we have resetRotation, did you mean storeAsInitial camera false?

Reply:
<img width="856" alt="image" src="https://github.com/cornerstonejs/cornerstone3D/assets/93064150/e659acd9-9838-4308-8da8-62733daba46a">

Reply:
<img width="531" alt="image" src="https://github.com/cornerstonejs/cornerstone3D/assets/93064150/32a020bc-9379-49b5-ab27-df3b79c624af">

Reply:
resetRotation goes through VolumeViewport, the first 3 go through the super

Reply:
but then the part you are touching is in rendering engine, and it is vtkJSDrivenViewports which is both Istack and I volume